app/static/pages/home/LearnerInfoDefinitions.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+oppiaFoundationWebsite.constant('LEARNER_INFO', [
+
+  {
```

Comment:
Shouldn't this include two different testimonials?

Reply:
The testimonials' text is kept as is from the mock.
@dchen97 do we have more testimonials somewhere?

Reply:
Here's another quote we can use from another learner in India - not certain on age though:
"I liked learning the lessons on mobile because we get very good questions and, if we get something wrong, we are told how to correct it. We are not scared when doing this, because we get hints too. I really liked this app."

Since these are both quotes from learners in India, we'll likely have to use different pictures as well to match. Here is a picture we can use instead to match the quotes:
![IMG_20171101_114358 (1)](https://user-images.githubusercontent.com/12034267/58705193-139cd100-8374-11e9-8883-cb994cff173d.jpg)

@seanlip - Do you have another picture we can use of our India learners?

Reply:
Yep -- I'll send some notes tonight. Stay tuned!

Reply:
Done :)